pkg/quarterdeck/db/migrations/0001_initial_schema.sql

```diff
@@ -56,6 +56,7 @@ CREATE TABLE IF NOT EXISTS api_keys (
     created_by          BLOB NOT NULL,
     source              TEXT DEFAULT NULL,
     user_agent          TEXT DEFAULT NULL,
+    partial             BOOL DEFAULT false,
```

Comment:
Note - this will require us to delete the quarterdeck database again next time we deploy.

Reply:
More of a note for me than anyone else ;-)

Reply:
Database deleted on deployment to staging.